dockerfiles/jupyter/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM jupyter/scipy-notebook
+
+# by default COPY creates files with root permissions, so doing all the preparation with root and chown-ing everything
+# in the end
+USER root
+COPY . /tmp/mlrun
+RUN cd /tmp/mlrun && pip install ".[api]" && cd /tmp && rm -rf mlrun
+
+WORKDIR $HOME
+
+COPY ./README.md $HOME
+COPY ./examples $HOME/examples/
+
+ARG MLRUN_CACHE_DATE=initial
+RUN git clone https://github.com/mlrun/demos.git $HOME/demos
+RUN git clone https://github.com/mlrun/functions.git $HOME/functions
+
+RUN mkdir data
+
+RUN chown -R $NB_UID:$NB_GID $HOME
+
+USER $NB_UID
+
+ENV MLRUN_DBPATH=http://localhost:8080
+ENV MLRUN_ARTIFACT_PATH=$HOME/data
+ENV JUPYTER_ENABLE_LAB=yes
+ENV MLRUN_HTTPDB__DATA_VOLUME=$HOME/data
+ENV MLRUN_HTTPDB__DSN='sqlite:////home/jovyan/data/mlrun.db?check_same_thread=false'
+
+CMD mlrun db & start-notebook.sh \
```

Comment:
is the single & here a mistake? was it supposed to be `&&`?

Reply:
no it's by purpose, `mlrun db` starts the mlrun api (yes, bad naming), so we want to start the api in the background